apps/vr-tests-react-components/src/stories/Rating.stories.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { Steps, StoryWright } from 'storywright';
+import { storiesOf } from '@storybook/react';
+import { CircleRegular, CircleFilled, SquareRegular, SquareFilled } from '@fluentui/react-icons';
+import { Rating } from '@fluentui/react-rating-preview';
+import { TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+
+storiesOf('Rating Converged Color', module)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <StoryWright
+      steps={new Steps()
+        .snapshot('rest', { cropTo: '.testWrapper' })
+        .hover('input')
+        .snapshot('hover', { cropTo: '.testWrapper' })
+        .hover('input'[4])
+        .snapshot('active', { cropTo: '.testWrapper' })
```

Comment:
This selector won't work as it is. I think `'input:nth-of-type(4)'` might work. But it's probably fine to just remove it.
```suggestion
```